docs/dense-retrieval.md

```diff
@@ -125,22 +125,24 @@ $ tools/eval/trec_eval.9.0.4/trec_eval -c -mrecall.1000 -mmap \
 map                     all     0.3702
 recall_1000             all     0.9734
 ```
+> To evaluate with on-the-fly query encoding, replace `--encoded-queries` with our pretrained encoder model
+> on [Hugging Face](https://huggingface.co/castorini/tct_colbert-msmarco/tree/main) 
+> `--encoder castorini/tct_colbert-msmarco`. The encoding will run on CPU by default. To enable GPU, add `--device cuda:0`.
 
 MS MARCO passage ranking task, 
 Hybrid
 - dense retrieval with TCT-ColBERT, brute force index.
 - sparse retrieval with doc2query-T5 expanded index.
 
 ```
-python -m pyserini.hsearch --topics msmarco_passage_dev_subset \
-                             --dense-index msmarco-passage-tct_colbert-bf \
-                             --sparse-index msmarco-passage-expanded \
-                             --encoded-queries msmarco-passage-dev-subset-tct_colbert \
-                             --output runs/run.msmarco-passage.tct_colbert.bf.doc2queryT5.tsv \
-                             --msmarco \
-                             --alpha 0.24 \
-                             --batch 36 \
-                             --threads 12
+python -m pyserini.hsearch   dense --index msmarco-passage-tct_colbert-bf \
```

Comment:
Also, comment I forgot to make from last time:

Is `--index msmarco-passage-tct_colbert-bf` and `--encoded-queries msmarco-passage-dev-subset-tct_colbert` both needed? As in, aren't the two options redundant? Would you ever want to use a "mismatched" set of queries? (Especially since we already have `--topics msmarco_passage_dev_subset`.)

How about just `--index` and `--encoder`? `--index` is required, `--encoder` is optional. If `--encoder` not specified, it uses pre-encoded queries. Since you know the index (and the topics), you know what pre-encoded queries you should use...

Would simply options a bit?

(This applies to dense retrieval alone also.)

Reply:
I'll update this in next PR, after integrating DPR. It may have some conflicts otherwise.